tests/callServer.spec.js

```diff
@@ -62,4 +62,12 @@ describe('CallServer', () => {
     driver.emit('call.started', ({callId, socket}));
   })
 
+  it('emits call.bridge event when the call has been bridged!', (done) => {
+    const callId = 1;
+    const socket = {};
+
+    callServer.on('call.bridged', done);
+    driver.emit('call.bridged', ({callId, socket}));
```

Comment:
This works ('cause the test does not verify the arguments emitted by the driver), but technically, the driver should only emit callId as the argument. We know that from here: https://github.com/utilitywarehouse/uw-lib-callcontrol/blob/answer-detection/src/callServer.js#L15. 
So:
```
driver.emit('call.bridged', callId);
```